src/General-Rules-Tests/ReTemporaryNeitherReadNorWrittenRuleTest.class.st

```diff
@@ -0,0 +1,31 @@
+Class {
+	#name : 'ReTemporaryNeitherReadNorWrittenRuleTest',
+	#superclass : 'ReAbstractRuleTestCase',
+	#category : 'General-Rules-Tests-Migrated',
+	#package : 'General-Rules-Tests',
+	#tag : 'Migrated'
+}
+
+{ #category : 'tests' }
+ReTemporaryNeitherReadNorWrittenRuleTest >> testRule [
+
```

Comment:
I wonder if the rule is triggered when we assign it and do not read it.

We should add a test checking that this is a violation in another PR:

`method |unused| unused := 1`